packages/suite/src/hooks/guide/useGuide.ts

```diff
@@ -11,12 +9,10 @@ export const useGuide = () => {
     const isGuideOpen = useSelector(state => state.guide.open);
     const dispatch = useDispatch();
 
-    const { layoutSize } = useLayoutSize();
+    const { isBelowLaptop } = useLayoutSize();
 
-    const isGuideOnTop = useMemo(
-        () => ['NORMAL', 'SMALL', 'TINY'].includes(layoutSize),
-        [layoutSize],
-    );
+    // The guide should be on top for smaller screens (below laptop size)
+    const isGuideOnTop = isBelowLaptop;
```

Comment:
nit: I suggest `isGuideOverlayContent`

Why: I had to test it in the app because I though it does overlay content all the time. This seems to me more descriptive what's actually hapening. Up to you.